steps/property-inspection/alterations.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  FieldsetLegend,
+  Heading,
+  HeadingLevels,
+} from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+} from "remultiform/component-wrapper";
+import { makeSubmit } from "../../components/makeSubmit";
+import {
+  PostVisitActionInput,
+  PostVisitActionInputProps,
+} from "../../components/PostVisitActionInput";
+import { RadioButtons } from "../../components/RadioButtons";
+import { ReviewNotes } from "../../components/ReviewNotes";
+import { getRadioLabelFromValue } from "../../helpers/getRadioLabelFromValue";
+import keyFromSlug from "../../helpers/keyFromSlug";
+import ProcessStepDefinition from "../../helpers/ProcessStepDefinition";
+import yesNoRadios from "../../helpers/yesNoRadios";
+import { Notes } from "../../storage/DatabaseSchema";
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import PageSlugs from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const questions = {
+  "has-alterations": "Have any alterations been made to the property?",
+};
+
+const step: ProcessStepDefinition<ProcessDatabaseSchema, "property"> = {
+  title: PageTitles.Alterations,
+  heading: "",
+  review: {
+    rows: [
+      {
+        label: questions["has-alterations"],
+        values: {
+          "has-alterations": {
+            renderValue(hasAlterations: string): React.ReactNode {
+              return getRadioLabelFromValue(yesNoRadios, hasAlterations);
+            },
+          },
+          "alterations-notes": {
+            renderValue(notes: Notes): React.ReactNode {
+              if (notes.length === 0) {
+                return;
+              }
+
+              return <ReviewNotes notes={notes} />;
+            },
+          },
+        },
+      },
+    ],
+  },
+  step: {
+    slug: PageSlugs.Alterations,
+    nextSlug: PageSlugs.Garden,
+    submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
+      makeSubmit({
+        slug: nextSlug as PageSlugs | undefined,
+        value: "Save and continue",
+      }),
+    componentWrappers: [
+      ComponentWrapper.wrapDynamic(
```

Comment:
It looks like you are missing the two paragraphs before this first question 
![image](https://user-images.githubusercontent.com/64883/89429877-00cf1a80-d736-11ea-82c6-cd62f0637a7f.png)